qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -154,7 +154,23 @@
           "url"
         ]
       },
-      "buttons": [],
+      "buttons": [
+        {
+          "enabled": true,
+          "type": "manifest",
+          "title": "Download Manifest",
```

Comment:
maybe the Datasets tab doesn't need a Download button... users might think they can download the datasets but it's just a csv of dataset metadata :/